Roman number to decimal

```diff
@@ -0,0 +1,40 @@
+
+def value(r):
```

Comment:
Use switch statement, they are far better instead of this.

Reply:
Yea